### ISSUE TITLE
[seafile-applet] segfault on libQt5Core.so.5.4.0

### ISSUE THREAD

**Issue by github-kiven** [haiwen/seafile#1136]
Hi, 
I have a segfault when I want to launch the applet on a Fedora 21 (segfault at 40 ip 0000003384cbb2d3 sp 00007ffd0878d7f0 error 4 in libQt5Core.so.5.4.0[3384a00000+4c2000]).

**Comment by Chilledheart** [comment]
If you are using the vendor's Qt5 libraries, i.e. the Qt5 libraries shipped with fedora, I can't tell much about it. In fact, I faced some memory issues (notably, bad aligns) with them, and once I switched to [the binary provided by Qt](http://download.qt.io/official_releases/qt/5.4/5.4.1/), they were gone. 

Sadly, I don't have time to do more future study on it. It is better to report it to the maintainers of your seafile repository if you are using any of them and [the maintainers of Qt5 libraries for fedora](https://admin.fedoraproject.org/pkgdb/package/qt5-qtbase/) as well.

In short, it is likely the vendor's problem.

**Comment by github-kiven** [comment]
Ok thanks for you quick answer, I will open a bug in Fedora.

**Comment by github-kiven** [comment]
just FYI, this repo works better 
https://copr.fedoraproject.org/coprs/logocomune/seafile-client/
